squad/frontend/templates/squad/build_settings.jinja2

```diff
@@ -0,0 +1,51 @@
+{% extends "squad/base.jinja2" %}
+
+{% block content %}
+<div ng-app='Build'>
+{% include "squad/build-nav.jinja2" %}
+
+  <h2>Build settings</h2>
+
+  <div ng-controller="BuildSettingsController">
+    {% raw %}
+    <div id="build-settings-alert" class="highlight-row">
```

Comment:
The intention was that the alert box would change after success, mainly because if the user changes 'keep data' again, the alert message will not change. I've moved the text to the save button (it shows depending of form_change) and now the success message slides up after 3 seconds. I can remove the animation if that's too distracting as well.

Reply:
I would add this animation to the side of the save button so it doesn't move the rest of the form vertically. having stuff shifting position by themselves in the page is very annoying

Reply:
@terceiro could you please check it again? I've moved the alert box next to the 'save' button and instead of sliding up, the alert box fades out, not moving anything else.

Reply:
nice, thanks for that.

I still have a few issues:

* move the checkbox to before the text
* align the save button to the left with the rest of the form

![screenshot_2019-02-27 squad -](https://user-images.githubusercontent.com/30027/53521912-0f80bf00-3ab8-11e9-879d-59312b1f3141.png)

also, if I change the form and save a second time, the green alert will not be displayed

Reply:
Hmm... It wasn't getting displayed back again only if you clicked the "x" button, which deleted the button tag. So instead of finding and overwriting bootstrap's behavior, I just deleted the "x" :), hope you're OK with that.

Also I added a few inline CSS, not the best practice, but I think if we keep getting more style changes it'd be worth to have a separate style file.